src/global/global.h

```diff
@@ -50,15 +50,6 @@ typedef double Real;
 
 #define LOG_FILE_NAME "run_output.log"
 
-// Conserved Floor Values
-#if (defined(FEEDBACK) || defined(STAR_FORMATION)) && defined(CLOUDY_COOL)
-  #define TEMP_FLOOR 1e1    // 10K for cloudy cooling
-  #define DENS_FLOOR 14.83  // 1e-6 cm^-3
-#else
-  #define TEMP_FLOOR 1e-3
-  #define DENS_FLOOR 1e-5  // in code units
-#endif
-
```

Comment:
Are these floors defined elsewhere now?

Reply:
They aren't used anymore.

I'm pretty sure they haven't been used ever since @helenarichie made the choices of floors into runtime parameters